app/form_objects/account/organisation_form.rb

```diff
@@ -0,0 +1,17 @@
+class Account::OrganisationForm < BaseForm
```

Comment:
I think this could be a prior commit with the corresponding spec.

Reply:
something to consider for other features?

I'd rather get this merged and move on to other work than break the existing commits up.